src/main/java/frc/robot/subsystems/gripper/GripperIOSim.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems.gripper;
+
+import static edu.wpi.first.units.Units.Radians;
+import static frc.robot.subsystems.gripper.GripperConstants.*;
+
+import com.ctre.phoenix6.controls.DutyCycleOut;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.Timer;
+import lib.motors.TalonFXSim;
+
+public class GripperIOSim implements GripperIO {
+    private final TalonFXSim rollerMotor;
+    private final TalonFXSim angleMotor;
+
+    private final DutyCycleOut powerRequest = new DutyCycleOut(0);
+    private final PositionVoltage positionRequest = new PositionVoltage(0);
+
+    public static PIDController angleController = new PIDController(KP.get(), KI.get(), KD.get());
+
+    public GripperIOSim() {
+        angleMotor =
+                new TalonFXSim(
+                        1,
+                        GripperConstants.ANGLE_MOTOR_GEAR_RATIO,
+                        0.5,
+                        GripperConstants.ANGLE_MOTOR_GEAR_RATIO);
+
+        rollerMotor = new TalonFXSim(1, 1, 0.5, 1);
+
+        angleMotor.setController(angleController);
+    }
+
+    @Override
+    public void setRollerMotorPower(double power) {
+        powerRequest.withOutput(power);
+        rollerMotor.setControl(powerRequest);
```

Comment:
All the `.with*` methods return the object, so you combine both lines:
```suggestion
        rollerMotor.setControl(powerRequest.withOutput(power));
```

Reply:
855bf1d768d6b98af4dcefe8fbafa400ab70426a